Algorithms/Search/JumpSearcher.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+namespace Algorithms.Search {
+
+    /// <summary>
+    /// The idea: This is one of the public search algorithm.
+    /// It takes 2 argument array of numbers and value of number
+    /// It finds index of this value in given array
+    ///     Time compl is O(root(n))
+    /// Note: the array has to be sorted beforehand.
+    /// </summary>
+    public class JumpSearcher {
```

Comment:
Take a look at https://github.com/TheAlgorithms/C-Sharp/blob/master/Algorithms/Search/BinarySearcher.cs
Could this searcher be implemented to work with IComparable<T>?

Reply:
Good point. I totally forgot to implement it.